packages/elements-core/src/components/Docs/HttpOperation/Body.tsx

```diff
@@ -23,7 +24,7 @@ export const isBodyEmpty = (body?: BodyProps['body']) => {
   return contents.length === 0 && !description?.trim();
 };
 
-export const Body = ({ body, onChange }: BodyProps) => {
+export const Body = ({ body, onChange, renderExtensionAddon }: BodyProps) => {
   const [refResolver, maxRefDepth] = useSchemaInlineRefResolver();
   const [chosenContent, setChosenContent] = React.useState(0);
   const { nodeHasChanged } = useOptionsCtx();
```

Comment:
Thank you @brendarearden. I hope to pick up end of next week and make the requested changes.

I also would like to introduce the functionality to expand/collapse all properties of a schema. Only I don't know yet what the best approach to achieve this.

Reply:
I believe you would be able to utilize the `expandedDepth` from the `SchemaTreeOptions` in json-schema-viewer to achieve that functionality. If you do decide to introduce that, it would be preferable to have a separate PR from this one.

Reply:
Yeah, definitely in a new PR for that one. I think the main issue I saw that the expansion state is stored  in `RowSchemaRow`-component but probably better to discuss the `json-schema-viewer`-repo first. Probably worth to add the `expandAll/collapseAll`-functionality there first.

Reply:
hey @weyert are you planning on continuing this work?

Reply:
Next week 👍

Reply:
@daniel-white Am I blocking you?

Reply:
@daniel-white @brendarearden  Would be fine to import `SchemaNode` from the `@stoplight/json-schema-tree`-package?